workloads/router-perf/run_router_test.sh

```diff
@@ -43,7 +43,7 @@ fi
 echo "Starting test for: $HTTP_TEST_SUFFIX"
 rm -rf /tmp/workloads
 git clone http://github.com/openshift-scale/workloads /tmp/workloads
-echo "[orchestration]" > /tmp/workloads/inventory; echo "${ORCHESTRATION_HOST:-localhost}" >> /tmp/workloads/inventory
+echo "[orchestration]" > /tmp/workloads/inventory; echo "${ORCHESTRATION_HOST:-localhost} ansible_user=${ORCHESTRATION_USER:-$USER}" >> /tmp/workloads/inventory
```

Comment:
Maybe set root as the default user if not defined?

Reply:
done